testsuite/sequential_execution_performance.py

```diff
@@ -31,9 +32,12 @@
 )
 
 # Check if any threshold is not met
-if tps_1k < THRESHOLD_1k or tps_10k < THRESHOLD_10k or tps_50k < THRESHOLD_50k:
-    print("Sequential TPS below the threshold")
+if (
+    abs(tps_1k - THRESHOLD_1k) > THRESHOLD_NOISE * THRESHOLD_1k
+    or abs(tps_10k - THRESHOLD_10k) > THRESHOLD_NOISE * THRESHOLD_10k
+    or abs(tps_50k - THRESHOLD_50k) > THRESHOLD_NOISE * THRESHOLD_50k
+):
+    print(
+        f"Sequential TPS below or above the threshold by {THRESHOLD_NOISE * 100}%! Please optimize the execution performance if below threshold, otherwise please increase the hard-coded threshold if you improved the execution performance. :)"
```

Comment:
instead of if being in the comment, can we print a different message whether we were above or below the threshold?